data/autoyast_sle15/autoyast_containers_ppc64le.xml

```diff
@@ -117,6 +117,12 @@
     <packages config:type="list">
       <package>iputils</package>
       <package>sles-release</package>
+      <package>runc</package>
+      <package>docker</package>
+      <package>docker-runc</package>
+      <package>zypper-docker</package>
+      <package>podman</package>
+      <package>podman-cni-config</package>
       <package>sle-module-server-applications-release</package>
```

Comment:
same goes to this guys , I would say that they should be autoselected when you choose some module , right ?

Reply:
I can do a VR removing these lines

Reply:
let's do one , but in case it will fail . I am fine to just merge it as it . don't think we need to invest a lot of time ...